### PR TITLE
README: Header redesign

### PR DESCRIPTION
**Pull request by liamcharger**
This PR enhances the visual appeal of the main README.md header with a restyled header design. Open to feedback and suggestions for further improvements!

Additionally, in the README body, I've included a reference to [InfiniTimeExplorer](https://infinitimeexplorer.netlify.app/) since its existence is not widely known.

Please refer to [my InfiniTime fork](https://github.com/liamcharger/InfiniTime) for a preview of the changes.

### CODE REVIEW

**Comment by rambonette** [comment]
Looks very nice imo 👏

**Review by NeroBurner — APPROVED** [review]
this looks great! I didn't know you could combine html tags with markdown

I especially like the collage. I'm curious, how did you create it?

**Comment by liamcharger** [comment]
> this looks great! I didn't know you could combine html tags with markdown
> 
> 
> 
> I especially like the collage. I'm curious, how did you create it?

Glad you like it! I did it in Adobe Photoshop by taking several screenshots from InfiniSim ;) and then merging each screen with a representation of the physical watch that was used in InfiniLink. Then I positioned everything, added the screen that's stacked on the others, and lastly, I added a circle with a Gaussian blur behind everything for the shadow effect.

**Comment by tituscmd** [comment]
This is beautiful. Great work!

File: README.md [hunk 1]
```diff
@@ -22,8 +38,13 @@ Fast open-source firmware for the [PineTime smartwatch](https://pine64.org/devic
 - [InfiniLink](https://github.com/InfiniTimeOrg/InfiniLink) (iOS)
 - [ITD](https://gitea.elara.ws/Elara6331/itd) (Linux)
 - [WatchMate](https://github.com/azymohliad/watchmate) (Linux)
+- [InfiniTimeExplorer](https://infinitimeexplorer.netlify.app) (Web)
+
+<br>
 
-***Note**: We removed mentions to NRFConnect as this app is closed source and recent versions do not work anymore with InfiniTime (the last version known to work is 4.24.3). If you used NRFConnect in the past, we recommend you switch to [Gadgetbridge](https://gadgetbridge.org/).*
+> *InfiniTimeExplorer is only compatible with web browsers that support Web BLE. Currently supported browsers include Chrome, Edge, and Firefox.*
```
Comment (mark9064) [inline README.md:45]: I could be misreading https://developer.mozilla.org/en-US/docs/Web/API/Web_Bluetooth_API, but it looks like firefox is not supported

**Review by mark9064 — COMMENTED** [review]
Looks really pretty :)

**Comment by FintasticMan** [comment]
I like the way it looks, but I do find it a bit strange that the yellow in the picture is darker than it "should" be.

**Comment by liamcharger** [comment]
> I do find it a bit strange that the yellow in the picture is darker than it "should" be.

Not sure if it really matters, but do you mean the yellow in the InfiniTime logo?

**Comment by FintasticMan** [comment]
Yeah, the yellow in the InfiniTime logo is darker than other places we use it. It looks like the colours are inverted actually, so I can see how it happened. I feel like maybe you could use the picture we use for the recovery firmware, that's on a black background.

**Comment by FintasticMan** [comment]
@liamcharger I think you accidentally pushed an unrelated change to this branch. We recommend using separate branches for each PR, and not using main, to prevent this happening by accident.

**Comment by liamcharger** [comment]
Oops, that was an accident...thought I was on a different branch.

**Comment by liamcharger** [comment]
> I feel like maybe you could use the picture we use for the recovery firmware, that's on a black background.

@FintasticMan do you know where I can get an image resource of the recovery firmware picture?

**Review by mark9064 — COMMENTED** [review]
Compare the colour of the centre picture to what's currently on the README
image

I'm assuming Fin means this?

**Comment by liamcharger** [comment]
Ok, I'll do that.

**Comment by liamcharger** [comment]
@FintasticMan @mark9064 @JF002 all done!

**Review by FintasticMan — APPROVED** [review]
Looks good to me! If you update it to the latest main I'll merge it.

**Comment by liamcharger** [comment]
Great! The branch is already up to date, though.

**Comment by FintasticMan** [comment]
Ah, sorry, I had rebase merge selected rather than squash merge.

**Pull request merged**